testnet/install/mac.md

```diff
@@ -18,18 +18,22 @@ Tested on MacOS Sequoia Version 15.3.1 (Apple Silicon)
 
 Getting started with the Zond testnet Local system
 
-## 1. Install prerequisits 
+## 1. Install prerequisites 
 
-If you haven't done so already, install brew, then install [docker](https://docs.docker.com/desktop/setup/install/mac-install/) and other prerequesits with brew.
+If you haven't done so already, install brew, then install [docker](https://docs.docker.com/desktop/setup/install/mac-install/) and other prerequesites with brew.
 
 ```bash
 brew install bazel@7 kurtosis-tech/tap/kurtosis-cli jq yq
 ```
 
-Be sure to add bazel 7.5.0 to your path.
+Run this to make bazel 7 the default version (if other versions are installed)
+```bash
+brew unlink bazel@8 && brew link --force bazel@7
+```
 
+Be sure to add bazel 7.5.0 to your default shell's path
 ```bash
-echo 'export PATH="/opt/homebrew/opt/bazel@7/bin:$PATH"' >> ~/.zshrc
+echo 'export PATH="/opt/homebrew/opt/bazel@7/bin:$PATH"' >> ~/.${SHELL##*/}rc

```

Comment:
Not all mac users will have zsh as their SHELL (I set mine to bash). This takes your shell
```bash
$ echo $SHELL
/bin/bash
```
and `${SHELL##*/}` yields `bash`, followed by concatting rc to refer to the `~/.bashrc` or `~/.zshrc` file for sending in the updated PATH.

Reply:
bazelisk avoids going down this rabbit hole